extensions/amp-analytics/amp-video-analytics.md

```diff
@@ -0,0 +1,142 @@
+}
+```
+
+### Video paused trigger (`"on": "video-paused"`)
+
+The `video-paused` trigger is fired when the video stops playing from a user clicking pause, from autoplay pausing, or from the video reaching the end. Use these configurations to fire a request for this event.
+
+```javascript
+"triggers": {
+  "myVideoPaused": {
+    "on": "video-paused",
+    "request": "event"
+  },
+  "videoSpec": {/* optional videoSpec */}
+}
+```
+
+### Video ended trigger (`"on": "video-ended"`)
+
+The `video-ended` trigger is fired when the video has reached the end of playback. Use these configurations to fire a request for this event.
+
+```javascript
+"triggers": {
+  "myVideoEnded": {
+    "on": "video-ended",
+    "request": "event"
+  },
+  "videoSpec": {/* optional videoSpec */}
+}
+```
+
+### Video session trigger (`"on": "video-session"`)
+
+The `video-session` trigger is fired when a "video session" has ended. A video session starts when a video is played and ends when video pauses, ends, or goes out of the viewport. This trigger is configurable in the `videoSpec` by setting `end-session-when-invisible`. Autoplaying videos can be optionally removed from this reporting by setting `exclude-autoplay` in the `videoSpec`. Use these configurations to fire a request for this event.
+
+```javascript
+"triggers": {
+  "myVideoSession": {
+    "on": "video-session",
+    "request": "event"
+  },
+  "videoSpec": {/* optional videoSpec */}
+}
+```
+
+### Video seconds played trigger (`"on": "video-seconds-played"`)
+
+The `video-seconds-played` trigger is fired when a specified number of seconds have passed since the last time the event fired. This *requires* `interval` to be set in the `videoSpec`. Use these configurations to fire a request for this event.
+
+```javascript
+"triggers": {
+  "myVideoSecondsPlayed": {
+    "on": "video-seconds-played",
+    "request": "event"
+  },
+  "videoSpec": {
+    "interval": 10, /* required */
+    /* other optional videoSpec properties */
+  }
+}
+```
+
+
+## Video spec
+
+You can use the `videoSpec` configuration object to control how the requests are fired.
+
+### `end-session-when-invisible`
+
+This spec property controls the `video-session` trigger. If true, the trigger will fire when the video is no longer visible, even if it is still playing. The default value is `false`.
+
+### `exclude-autoplay`
+
+This spec property controls all `video-*` triggers. If true, the trigger will not fire for videos that are currently autoplaying, not just initially autoplaying. If a user interacts with the video and changes its state from autoplaying, the trigger will include events issued by the video. The default value is `false`.
+
+### `interval`
+
+This spec property controls the `video-seconds-played` trigger, and it is *required* for this trigger. Its value is the number of seconds between each `video-seconds-played` event. It must be greater than `0`. For example, a value of `10` would fire the trigger every 10 seconds.
```

Comment:
- rewrite "Its value is the number of seconds between each `video-seconds-played` event. It must be greater than `0`. " as "The value for `interval` specifies the number of seconds between each `video-seconds-played` event, and it must be greater than `0`.
- replace "would fire" to "fires"